clients/fides-js/rollup.config.mjs

```diff
@@ -21,70 +25,91 @@ const preactAliases = {
   ],
 };
 
+const fidesScriptPlugins = ({ name, gzipWarnSizeKb, gzipErrorSizeKb }) => [
+  alias(preactAliases),
+  nodeResolve(),
+  postcss({
+    minimize: !IS_DEV,
+  }),
+  esbuild({
+    minify: !IS_DEV,
+  }),
+  copy({
+    // Automatically add the built script to the privacy center's static files for bundling:
+    targets: [
+      { src: `dist/${name}.js`, dest: "../privacy-center/public/lib/" },
+    ],
+    verbose: true,
+    hook: "writeBundle",
+  }),
+  filesize({
+    reporter: [
+      "boxen", // default reporter, which prints a nice CLI output
+
+      // Add a defensive check to fail the build if our bundle size starts getting too big!
+      (options, bundle, { gzipSize, fileName }) => {
+        const gzipSizeKb = Number(gzipSize.replace(" KB", ""));
+        if (gzipSizeKb > gzipErrorSizeKb) {
+          console.error(
+            `❌ ERROR: ${fileName} build failed! Gzipped size (${gzipSize}) exceeded maximum size (${gzipErrorSizeKb} KB)!`,
+            `If you must, update GZIP_SIZE_* constants in clients/fides-js/rollup.config.mjs`
+          );
+          process.exit(1);
+        } else if (gzipSizeKb > gzipWarnSizeKb) {
+          console.warn(
+            `️🚨 WARN: ${fileName} build is getting large! Gzipped size (${gzipSize}) exceeded warning size (${gzipWarnSizeKb} KB)!`,
+            `If you must, update GZIP_SIZE_* constants in clients/fides-js/rollup.config.mjs`
+          );
+          if (IS_DEV) {
+            process.exit(1);
+          }
+        } else {
+          console.log(
+            `✅ ${fileName} gzipped size passed maximum size checks (${gzipSize} < ${gzipErrorSizeKb} KB)`
+          );
+        }
+      },
+    ],
+  }),
+];
+
+const SCRIPTS = [
```

Comment:
refactored so that we didn't have to copy/paste as much. in the future, we can add more scripts to this array, and add whatever props we need to make the scripts work 👍